### PR TITLE
style changes

### PR DESCRIPTION
**Pull request by dmercer-google**
Some style changes based on Google internal build analysis

### CODE REVIEW

File: src/com/makina/security/OpenFIPS201/ECParamsP256.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 final class ECParamsP256 extends ECParams {
 
   // Curve polynomial element a
-  private static final byte[] a = {
+  private static byte[] a = {
```
Comment (mistial-dev) [inline src/com/makina/security/OpenFIPS201/ECParamsP256.java:36]: Why wouldn't these be final?

Comment (mistial-dev) [reply src/com/makina/security/OpenFIPS201/ECParamsP256.java:36]: Looking at the Google style guide,

"Constants are static final fields whose contents are deeply immutable and whose methods have no detectable side effects. Examples include primitives, strings, immutable value classes, and anything set to null. If any of the instance's observable state can change, it is not a constant. Merely intending to never mutate the object is not enough."

It's static, and I'd think it's a constant with a deeply immutable nature.

File: src/com/makina/security/OpenFIPS201/ECParamsP384.java [hunk 2]
```diff
@@ -76,7 +76,7 @@ public static ECParams Instance() {
   };
 
   // Base point
-  public static final byte[] G = {
```
Comment (mistial-dev) [inline src/com/makina/security/OpenFIPS201/ECParamsP384.java:79]: As a constant, should this not be G?

File: src/com/makina/security/OpenFIPS201/ECParamsP384.java [hunk 3]
```diff
@@ -163,6 +163,6 @@ protected byte[] getN() {
 
   @Override
   protected short getH() {
-    return h;
+    return H;
```
Comment (mistial-dev) [inline src/com/makina/security/OpenFIPS201/ECParamsP384.java:166]: Why is `h` turned to H, but G turned to `g`?

File: src/com/makina/security/OpenFIPS201/PIV.java [hunk 4]
```diff
@@ -1758,10 +1764,9 @@ public void putDataAdmin(byte[] buffer, short offset, short length) {
   public void changeReferenceDataAdmin(byte id, byte[] buffer, short offset, short length) {
 
     final byte CONST_TAG_SEQUENCE = (byte) 0x30;
-    final byte CONST_TAG_KEY = (byte) 0x80;
-    final byte CONST_TAG_RSA_N = (byte) 0x81; // RSA Modulus
-    final byte CONST_TAG_RSA_E = (byte) 0x82; // RSA Public Exponent
-    final byte CONST_TAG_RSA_D = (byte) 0x83; // RSA Private Exponent
+    // RSA Modulus
```
Comment (mistial-dev) [inline src/com/makina/security/OpenFIPS201/PIV.java:1767]: Do these comments make sense if you remove the code they refer to?

File: src/com/makina/security/OpenFIPS201/PIVKeyObjectECC.java [hunk 5]
```diff
@@ -249,27 +255,27 @@ private void setPrivateParams() {
...
+    ((ECPrivateKey) privateKey).setB(b, (short) 0, (short) b.length);
+    ((ECPrivateKey) privateKey).setG(g, (short) 0, (short) g.length);
+    ((ECPrivateKey) privateKey).setR(r, (short) 0, (short) r.length);
+    ((ECPrivateKey) privateKey).setFieldFP(p, (short) 0, (short) p.length);
     ((ECPrivateKey) privateKey).setK(params.getH());
   }
 
   /** Set ECC domain parameters. */
-  protected void setPublicParams() {
+  void setPublicParams() {
```
Comment (mistial-dev) [inline src/com/makina/security/OpenFIPS201/PIVKeyObjectECC.java:267]: Is there an issue with this being protected?

File: src/com/makina/security/OpenFIPS201/PIVKeyObjectSYM.java [hunk 6]
```diff
@@ -95,7 +95,7 @@ public void updateElement(byte element, byte[] buffer, short offset, short lengt
     PIVSecurityProvider.zeroise(buffer, offset, keyLengthBytes);
   }
 
-  protected void allocate() {
+  void allocate() {
```
Comment (mistial-dev) [inline src/com/makina/security/OpenFIPS201/PIVKeyObjectSYM.java:98]: The style guide doesn't seem to have an issue with protected.

**Review by mistial-dev — COMMENTED** [review]
In general, seems to be cleaner code.

It removes some things that might be intended for later functionality, and there are a few case inconsistencies.  I'm curious why so many final things were made not final.

File: src/com/makina/security/OpenFIPS201/ECParamsP256.java (hunk 1 above)
Comment (makinako) [reply src/com/makina/security/OpenFIPS201/ECParamsP256.java:36]: This is an old pull request that is too far de-sync'd with the master branch now. You can see in the [head revision](https://github.com/makinako/OpenFIPS201/blob/1cdc610ab91379801eedaf804139af82a1d4c383/src/com/makina/security/openfips201/ECParamsP256.java#L36) these are all private static final.

**Pull request closed without merging**